app/config/api.py

```diff
@@ -29,35 +38,75 @@ def validation_error_to_response(request: HttpRequest, exception: ValidationErro
     messages = extract_error_messages(exception)
     return api.create_response(
         request,
-        {"detail": messages},
+        {
+            "code": status, "description": messages
+        },
         status=status,
     )
 
 
 @api.exception_handler(Http404)
-def http404_to_response(request: HttpRequest, exception: Http404) -> HttpResponse:
-    """Convert the given exception to a Not Found response with the actual exception message.
-
-    Without this, we would only see "Not found" in the response even if the
-    original exception message is more detailed like "No User matches the given query.".
-    """
+def handle_404_not_found(request: HttpRequest, exception: Http404) -> HttpResponse:
     return api.create_response(
         request,
-        {"detail": str(exception)},
+        {
+            "code": 404, "description": "Resource not found"
```

Comment:
Do we need the `"code"` if we already have it in the `status`?

Reply:
I've taken this from the API (draft) specs. I general, I think it is a good idea to also include the status code in the response body, as it can make my live as a (programming) user of this API easier.

Reply:
I don't see the advantage but, if that is common practice, fine with keeping it.

Reply:
Let's discuss and make a decision today.

Reply:
As discussed today: We'll use the same response structure as with [STAC](https://data.geo.admin.ch/api/stac/static/spec/v0.9/api.html#tag/Data/operation/getCollections): code and description